manager/src/main/java/org/openremote/manager/setup/builtin/BuiltinSetupTasks.java

```diff
@@ -41,6 +41,7 @@
             addTask(new ManagerCleanSetup(container));
             addTask(new KeycloakCleanSetup(container));
             addTask(new KeycloakInitSetup(container));
+            addTask(new ArtNetDemoSetup());
```

Comment:
Don't want a fixed setup in our system for `Artnet` for reasons mentioned in `ManagerDemoSetup`